src/test/clojure/xtdb/pgwire_test.clj

```diff
@@ -865,11 +865,11 @@
       (is (= [{:a {"b" 42}}] rs)))))
 
 (deftest start-stop-as-module-test
```

Comment:
Only place we actually test pgwire inside the node - lots of other stuff in this module, not entirely related to configuration, but worth a think about.